src/config/index.ts

```diff
@@ -213,6 +214,7 @@ export const UPDATE_REVIEW_OVERSEAS_ENTITY_INFORMATION_URL = UPDATE_AN_OVERSEAS_
 export const UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + UPDATE_DO_YOU_WANT_TO_MAKE_OE_CHANGE_PAGE;
 export const UPDATE_NO_CHANGE_BENEFICIAL_OWNER_STATEMENTS_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + UPDATE_NO_CHANGE_BENEFICIAL_OWNER_STATEMENTS_PAGE;
 export const UPDATE_NO_CHANGE_REGISTRABLE_BENEFICIAL_OWNER_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + UPDATE_NO_CHANGE_REGISTRABLE_BENEFICIAL_OWNER_PAGE;
+export const UPDATE_REVIEW_STATEMENT_BEFORE_SUBMITTING_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + UPDATE_REVIEW_STATEMENT_PAGE;
```

Comment:
We could just keep the naming similar here
```suggestion
export const UPDATE_REVIEW_STATEMENT_URL = UPDATE_AN_OVERSEAS_ENTITY_URL + UPDATE_REVIEW_STATEMENT_PAGE;
```

Reply:
Renamed